warehouse/integrations/bigquery/bigquery.go

```diff
@@ -735,7 +733,7 @@ func (bq *HandleT) CrashRecover(warehouse model.Warehouse) (err error) {
 	return
 }
 
-func (bq *HandleT) dropDanglingStagingTables() bool {
+func (bq *BigQuery) dropDanglingStagingTables() bool {
```

Comment:
I see a lot of `Info` & `Error` log in this func. Can't we get rid of error log by return it to caller `CrashRecover()` and are that many `Info` logs really required?
```suggestion
func (bq *BigQuery) dropDanglingStagingTables() error {
```

Reply:
Certainly.

Reply:
Will take it as a separate task.
https://www.notion.so/rudderstacks/BigQuery-drop-table-cleanup-e1aa63df73254a68858cf82e4e7eb792?pvs=4